internal/querynodev2/delegator/delegator_data.go

```diff
@@ -472,6 +472,12 @@ func (sd *shardDelegator) LoadSegments(ctx context.Context, req *querypb.LoadSeg
 	// alter distribution
 	sd.distribution.AddDistributions(entries...)
 
+	partStatsToReload := make([]UniqueID, 0)
+	lo.ForEach(req.GetInfos(), func(info *querypb.SegmentLoadInfo, _ int) {
+		partStatsToReload = append(partStatsToReload, info.PartitionID)
+	})
+	sd.maybeReloadPartitionStats(ctx, partStatsToReload...)
+
 	return nil
 }
```

Comment:
when loading or releasing segments, we try to update partition stats for the partitions that loaded segments belong to

Reply:
why release also needs update partition stats? For handoff?